src/commands/mod.rs

```diff
@@ -44,6 +45,7 @@ pub enum Command {
     /// can submit proposals (such as a Motion Proposal) to be voted on by other neuron
     /// holders.
     MakeProposal(make_proposal::MakeProposalOpts),
+    RefundSwap(refund_swap::RefundSwapOpts),
```

Comment:
Can you add some documentation?

Reply:
The documentation comment is inherited from the RefundSwapOpts, as described above.